### ISSUE TITLE
pydantic error at initialization of dataset

### ISSUE THREAD

**Issue by ovidozio** [CESNET/cesnet-datazoo#4]
I tried to load the dataset following the provided code to initialize the dataset.
Following for example `explore_data.ipynb` the code 

```{python}
from cesnet_datazoo.datasets import CESNET_QUIC22, CESNET_TLS22, CESNET_TLS_Year22
from cesnet_datazoo.config import DatasetConfig

dataset = CESNET_QUIC22(data_root="data/CESNET-QUIC22/", size="XS")
```

produces a fatal error

```
PydanticUndefinedAnnotation: name 'CesnetDataset' is not defined
```

**Comment by janluxemburk** [comment]
Hi, thanks for reporting this issue. There was a problem with the newest version of Pydantic. I fixed the version to `pydantic<=2.8.2`

There is a new cesnet-datazoo version 0.1.8. Recreate the venv to install the correct dependencies.

**Comment by ovidozio** [comment]
I can confirm it works now, thank you. 

I just uninstalled and reinstalled the `cesnet-datazoo` package using pip and I no longer have the same error.
